dev/build.example.testcontainers_fat/fat/src/com/ibm/ws/testcontainers/example/DockerfileTest.java

```diff
@@ -0,0 +1,103 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.testcontainers.example;
+
+import java.time.Duration;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.runner.RunWith;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.containers.SimpleLogConsumer;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import web.generic.ContainersTestServlet;
+
+/**
+ * Example test class showing how to setup a testcontainer that uses a custom dockerfile.
+ */
+@RunWith(FATRunner.class)
+public class DockerfileTest {
+
+    public static final String APP_NAME = "containerApp";
+
+    @Server("build.example.testcontainers")
+    @TestServlet(servlet = ContainersTestServlet.class, contextRoot = APP_NAME)
+    public static LibertyServer server;
+
+    public static final String POSTGRES_DB = "test";
+    public static final String POSTGRES_USER = "test";
+    public static final String POSTGRES_PASSWORD = "test";
+
```

Comment:
Does it make sense to add POSTGRES_PORT and POSTGRES_DOCKERHUB_IMAGE variables here as well?
```
public static final String POSTGRES_DOCKERHUB_IMAGE = "kyleaure/postgres-test-table:1.0";
public static final int POSTGRES_PORT = 5432;
```

Reply:
Port - yes I will add that
Image - probably not since it is only used once.